src/components/_charts/EthBtcChart.tsx

```diff
@@ -177,6 +177,7 @@ export const EthBtcChart: VFC<{ timeline: string; name: string }> = ({
         tickRotation: 0,
```

Comment:
Suggestion:
use tickRotation on mobile view for axisBottom, so that the contents can be read

<img width="307" alt="image" src="https://user-images.githubusercontent.com/43783037/202823527-884d4649-abf5-40c2-96cc-9bb09a77fd30.png">

Reply:
adding tickRotation somehow make a client error

Reply:
fixed by [fix: line chart tick values mobile friendly](https://github.com/strangelove-ventures/sommelier/pull/653/commits/e3bd0395a2d60cebe36373243095551cc3bf4e38) 
![CleanShot 2022-11-19 at 09 01 45](https://user-images.githubusercontent.com/39829726/202829156-41a1525f-602f-4e1a-a23c-0797ae82b9e9.gif)

Reply:
Looks good 👍